pkg/generators/models/testdata/cases/parameter_model/api.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Test
+
+paths:
+  "/foo":
+    parameters:
+      - name: param1
```

Comment:
Can you add a path parameter as well. Also ,how does it handle things like repeated  parameters (ie Array values)

Reply:
It is agnostic about path/query/header parameters. Can you give me an example of "repeated parameters"? If I understand it right, they should just have an array type in their schema spec which should just work like for all other array types

Reply:
It needs to handle https://swagger.io/docs/specification/serialization/#query the query style for understanding if the value is an slice type and parsing the value into slice

Reply:
RE those docs, I think we should update the README to comment that we will 
1. only support `simple` style path parameters, which are the default
   > simple – (default) comma-separated values. Corresponds to the {param_name} URI template.
2. for query parameters we will start by supporting the `style` and `explode` value we have agreed on with frontend, ie what we do in hub right now with _no support_ for object values, only primitives and arrays.

Reply:
I added an example with a array schema. The stype and explode values are only important for parsing and I'm not (yet) generating a method to parse these structs from http requests.

Reply:
@trusch just a quick followup, can we add a path parameter too? For example

```
"/foo/{id}":
```

Reply:
done